docs/develop/reference/aspect-lib/join-points/verify-aspect/index.md

```diff
@@ -193,3 +193,222 @@ sys.log('block.header.number' + ' ' + numberData.data.toString(10));
 | env.consensusParams.appVersion               | UintData        | Get the ABCI application version.                                                                                                                                                         |
 | aspect.id                                    | BytesData       | Returns current aspect id.                                                                                                                                                                |
 | aspect.version                               | UintData        | Returns current aspect version.                                                                                                                                                           |
+
+## To trigger
+
+To trigger the 'VerifyTx' join point, a customized verification transaction without a signature is necessary. The format
+of this transaction is as follows:
+
+```shell
+magic prefix + checksum(encodedData) + encodedData(validation data + raw calldata)
+```
+
+**Description：**
+> * 0xCAFECAFE serves as a stable value with a magical prefix.
+> * checksum(encodedData) represents a 4-byte checksum of the encoded data.
+> * encodedData is the result of ABI encoding, combining validation data and raw call data using `ABI.encode(
+    ValidationData, CallData)."
+
+Example:
+
+```shell
+0xCAFECAFE75bac07d00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000
+0000000000000000000000000c000000000000000000000000000000000000000000000000000000000000000557d7cfdeff97cbabdc4124c779d51
+bb2520cbdc769840c6fee4566e23c7c22ea2a3e071dc794283b70608de4566c9dc0fd5f1854d7032676996fd95284693ecad5b2eeca9864734af5131
+ed015c52cd1b5b3d9f841c000000000000000000000000000000000000000000000000000000000000000000000000000000000000241003e2d2000
+000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000
+```
+
+* 0xCAFECAFE75 is a magical prefix.
+* 75bac07d is 4-byte checksum of the encoded data.
+* The remaining bytes represent the result of ABI encoding for ValidationData and CallData.
+
+### Create customized verification transaction
```

Comment:
This part is too long, the validation data constructed here is for session key, we need to provide a minimum case here.